src/main/java/uk/gov/pay/api/model/CardPayment.java

```diff
@@ -84,6 +93,7 @@ public String toString() {
                 ", returnUrl='" + returnUrl + '\'' +
                 ", reference='" + reference + '\'' +
                 ", language='" + language.toString() + '\'' +
+                ", delayedCapture=" + (delayedCapture != null ? delayedCapture.toString() : "") +
```

Comment:
Shouldn’t need the `null` check here: now that PP-4105 is done, connector should always return `delayed_capture`